common/taskana-common-test/src/main/java/pro/taskana/common/test/config/DataSourceGenerator.java

```diff
@@ -144,18 +121,21 @@ static String getSchemaNameFromPropertiesObject(String propertiesFileName) {
       if (!propertiesFileIsComplete) {
         LOGGER.warn("propertiesFile {} is incomplete {}", propertiesFileName, warningMessage);
         LOGGER.warn("Using default Datasource for Test");
-        schemaName = "TASKANA";
+        schemaName = DEFAULT_SCHEMA_NAME;
       }
 
-    } catch (Exception e) {
-      LOGGER.warn("Caught Exception ", e);
+    } catch (FileNotFoundException e) {
+      LOGGER.warn("getSchemaNameFromPropertiesObject caught Exception ", e);
       LOGGER.warn("Using default schemaName for Test");
+    } catch (IOException e) {
+      LOGGER.warn("createDataSourceFromProperties caught Exception ", e);
+      LOGGER.warn("Using default Datasource for Test");
     }
 
     return schemaName;
   }
 
-  private static DataSource createDefaultDataSource() {
+  private static DataSource createDataSourceForH2() {
     String jdbcDriver = "org.h2.Driver";
     String jdbcUrl =
         "jdbc:h2:mem:taskana;IGNORECASE=TRUE;LOCK_MODE=0;"
```

Comment:
@Tristan2357 did you forget to remove this?

Reply:
it seems like it

i will create a pr